variables.tf

```diff
@@ -208,6 +226,12 @@ variable "intra_subnets" {
   default     = []
 }
 
+variable "firewall_subnets" {
+  description = "A list of firewall subnets inside the VPC"
+  type        = list(string)
+  default     = []
+}
```

Comment:
```
Error: error creating NetworkFirewall Firewall (blah-blah-firewall): InvalidRequestException: availability zone subnet(s) cannot be mapped more than one per availability zone in a firewall, parameter: [subnet-00000000000000001, subnet-00000000000000002]

  on ../../lib/vpc/main.tf line 130, in resource "aws_networkfirewall_firewall" "this":
 130: resource "aws_networkfirewall_firewall" "this" {
 ```

There is a hard requirement for `length(var.firewall_subnets) <= length(var.azs)` imposed by aws.  The rest of this module does not behave this way outside of when the firewall is enabled. (AKA you can have 6 public subnets and 3 azs and other configurations)

This should be documented.

Reply:
Added note in variable description, as well as main README to highlight this point